.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,6 +11,10 @@ assignees: ''
 
 Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
 
+Download and run checkstyle cli: https://checkstyle.org/cmdline.html#Download_and_Run
+
+Cli describes the problem better than 1,000 words.
+
 ```bash
 /var/tmp $ javac YOUR_FILE.java
 #[[MAKE SURE THERE IS NO OUTPUT]]
```

Comment:
This is not necessarily true; when compiling using preview features with the `--enable-preview` flag, `javac` will output something like:

```
Note: TestClass.java uses preview language features.
Note: Recompile with -Xlint:preview for details.
```
Perhaps just specify that the user must show successful compilation of the example?

Reply:
Done

Reply:
Resolved